Tasks/ExtractFiles/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "extractFiles",
+  "version": "1.0.0",
+  "description": "Extract Files Task",
+  "main": "extractFiles.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Microsoft/vsts-tasks.git"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/vsts-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
+  "dependencies": {
+    "collections": "^3.0.0"
```

Comment:
cool.  This is the right way to do this.  You might want to take the caret out because this means 3.x.x.  Might want to be explcit